test/createSession.spec.js

```diff
@@ -55,13 +55,20 @@ describe('createSession', () => {
         sessionId: 'fakeSessionId',
         token: 'fakeToken',
         onStreamsUpdated,
+        options: {
```

Comment:
@julien-meichelbeck Could you please remove the proxyUrl option from the test since it's in beta? Maybe test with connectionEventSurpressed?

Reply:
sure thing